subsys/bluetooth/host/conn.c

```diff
@@ -2122,6 +2120,44 @@ static void bt_conn_set_param_le(struct bt_conn *conn,
 	conn->le.timeout = param->timeout;
 }
 
+static bool create_param_validate(const struct bt_conn_le_create_param *param)
+{
+	/* Valid range from CONFIG_BT_CREATE_CONN_TIMEOUT Kconfig symbol */
+#if defined(CONFIG_BT_PRIVACY)
+	const s32_t timeout_max = K_SECONDS(CONFIG_BT_RPA_TIMEOUT);
+#else
+	const s32_t timeout_max = K_SECONDS(65535);
```

Comment:
This is now wrong since #23382 has been merged, i.e. these macros return `k_timeout_t` rather than `s32_t`